src/Storage/Storage/Common/Cmdlet/NewAzureStorageContext.cs

```diff
@@ -150,6 +150,7 @@ public class NewAzureStorageContext : AzureDataCmdlet
         public string StorageAccountKey { get; set; }
 
         private const string SasTokenHelpMessage = "Azure Storage SAS Token";
+        [CmdletParameterBreakingChange("SasToken", ChangeDescription = "SAS token created in context will not have the leading question mark ?")]
```

Comment:
I think the change is create storage context with a SAS token no matter it has leading "?" or not, the SAS token in created Storage Context properties `ConnectionString`, and `StorageAccount.Credentials` won't have leading ‘?’.
So the word might can be:
The SAS token in created Storage Context properties 'ConnectionString', and 'StorageAccount.Credentials' won't have leading ‘?’ in a future release.

Reply:
Done